docs/uma-tokenholders/approved-collateral-currencies.md

```diff
@@ -3,81 +3,92 @@ title: Approved collateral currencies
+|ALPHA| 900 | [UMIP-77](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-77.md) | [Ethereum](https://etherscan.io/token/0xa1faa113cbe53436df28ff0aee54275c13b40975)
+|MKR| 0.2 | [UMIP-77](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-77.md) | [Ethereum](https://etherscan.io/token/0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2)
+|REN| 1150 | [UMIP-77](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-77.md) | [Ethereum](https://etherscan.io/token/0x408e41876cccdc0f92210600ef50372656052a38)
+|CRV| 250 | [UMIP-77](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-77.md), [UMIP-113](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-113.md) | [Ethereum](https://etherscan.io/token/0xD533a949740bb3306d119CC777fa900bA034cd52), [Polygon](https://polygonscan.com/address/0x172370d5cd63279efa6d502dab29171933a610af)
+|RGT| 85 | [UMIP-77](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-77.md) | [Ethereum](https://etherscan.io/token/0xd291e7a03283640fdc51b121ac401383a46cc623)
+|NFTX| 7.5 | [UMIP-77](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-77.md) | [Ethereum](https://etherscan.io/token/0x87d73e916d7057945c9bcd8cdd94e42a6f47f776)
+|LON| 130 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0x0000000000095413afc295d19edeb1ad7b71c952)
+|MASK| 115 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0x69af81e73a73b40adf4f3d4223cd9b1ece623074)
+|BANK| 5.75 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0x24a6a37576377f63f194caa5f518a60f45b42921)
+|SFI| 1.3 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0xb753428af26e81097e7fd17f40c88aaa3e04902c)
+|VSP| 50 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0x1b40183efb4dd766f11bda7a7c3ad8982e998421)
+|FRAX| 400 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0x853d955acef822db058eb8505911ed77f175b99e)
+|DEXTF| 2250 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0x5F64Ab1544D28732F0A24F4713c2C8ec0dA089f0)
+|ORN| 75 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md) | [Ethereum](https://etherscan.io/token/0x0258f474786ddfd37abce6df6bbb1dd5dfc4434a)
+|BOND| 10 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md), [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-113.md) | [Ethereum](https://etherscan.io/token/0x0391d2021f89dc339f60fff84546ea23e337750f), [Polygon](https://polygonscan.com/token/0xa041544fe2be56cce31ebb69102b965e06aace80)
```

Comment:
There are a bunch of links that specify different umip numbers but then link to umip-113. Umip 113 didn't add any collateral and really shouldn't be included in this list at all.

For collateral currencies that are on polygon and ethereum, you shoul just link to the umips that actually added these. In most cases they will be the same umip, so you'll just need one link.

Also a lot of blue chip collateral types were added to polygon without an umip, so you wouldn't need a link for those. Bond is one of these